src/usage/indexing.md

```diff
@@ -150,3 +79,51 @@ WITH (options = $$
 segment.max_growing_segment_size = 200000
 $$);
 ```
+
+## Suggestions
+
+Selecting the appropriate indexing algorithm and settings is crucial for vector search performance. Below are recommendations tailored to various use cases.
+
+### Brute force (flat)
+If you're conducting a limited number of searches (between 1,000 and 10,000) or require precise results, the [brute force](/reference/indexing_options.html#options-for-table-flat) algorithm is a viable option.
+
+This algorithm sets the standard for other indexing methods. It retains vectors in their original form without compression and avoids additional overhead. While straightforward and user-friendly, it's not ideal for processing large datasets.
+
+In addition, you can use quantization to reduce memory usage. For a complete list of all options, please refer to the [documentation](/reference/indexing_options.html#options-for-table-quantization). Detailed information about quantization can also be found [there](/usage/quantization.html).
+
+### Inverted file index (IVF)
+
+IVF mainly uses the idea of inverted indexing to store the vectors `(id, vector)` under each cluster center. When querying a vector, it finds the nearest several centers and searches for the vectors under these centers respectively.
+
+Before you begin your search, it's necessary to train nlist cluster centers. You also have the option to set nsample for K-Means clustering; a higher value yields more precise clusters but increases training time. For additional settings, please consult the reference guide.
+
+Before searching, train `nlist` cluster centers using K-Means clustering with `nsample` points. A higher value yields more precise clustering but requires longer training time. For additional settings, please consult the [reference guide](/reference/indexing_options.html#options-for-table-ivf).
+
+### Hierarchical navigable small world graph (HNSW)
+
+HNSW merges the concepts of [skip list](https://brilliant.org/wiki/skip-lists/) with navigable small world (NSW) graphs, employing a hierarchical structure that features longer edges on higher layers for rapid searching and shorter edges on lower layers to enhance search precision. 
```

Comment:
Use Wikipedia links https://en.wikipedia.org/wiki/Skip_list since it has a higher score.

Reply:
sgtm